composer.json

```diff
@@ -5,16 +5,12 @@
     "description": "Rector upgrades rules for Hyperf.",
     "require": {
         "php": ">=8.0",
-        "rector/rector": "^0.15.12"
+        "rector/rector": "^1.0"
     },
     "require-dev": {
         "phpunit/phpunit": "^10.0",
         "phpstan/phpstan": "^1.8.2",
         "symplify/phpstan-rules": "^11.0",
-        "symplify/phpstan-extensions": "^11.0",
-        "symplify/easy-coding-standard": "^11.0",
```

Comment:
required when execute `composer fix-cs`

Reply:
removed that one by accident, I put it back